app/src/main/kotlin/com/github/gotify/init/InitializationActivity.kt

```diff
@@ -130,4 +134,23 @@ internal class InitializationActivity : AppCompatActivity() {
             .version
             .enqueue(Callback.callInUI(this, callback, errorCallback))
     }
+
+    private fun runWithNeededPermissions(action: () -> Unit) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                // Android 13 and above
+                runWithPermissions(
```

Comment:
Currently, if you deny everything, the app only shows the initialization scree. I think the prompt shouldn't be cancelable. As we require these permissions.
![image](https://user-images.githubusercontent.com/14895212/215066876-3077b9b7-ee56-4922-bcda-d16ba3c3915e.png)

Reply:
Fixed.